### PR TITLE
feat(publicca): generate library

### PR DESCRIPTION
**Pull request by scotthart**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/14257)

### CODE REVIEW

File: google/cloud/publicca/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Public Certificate Authority API C++ Client Library
+
+This directory contains an idiomatic C++ client library for the \[Public
+Certificate Authority API\]\[cloud-service-docs\].
```
Comment (dbolduc) [inline google/cloud/publicca/README.md:4]: Hmm.. what happened here?

The scaffolding looks fine to me:
https://github.com/googleapis/google-cloud-cpp/blob/f992e93a50b187b096c6148c8603cf732025c2fa/generator/internal/scaffold_generator.cc#L349-L350

But maybe it does not work with `checkers` anymore?

File: google/cloud/publicca/quickstart/README.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+set GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=%cd%\roots.pem
+```
+
+[authentication-quickstart]: https://cloud.google.com/docs/authentication/getting-started "Authentication Getting Started"
+[bazel-install]: https://docs.bazel.build/versions/main/install.html
+[choco-cmake-link]: https://chocolatey.org/packages/cmake
+[grpc-roots-pem-bug]: https://github.com/grpc/grpc/issues/16571
+[homebrew-cmake-link]: https://formulae.brew.sh/formula/cmake
+[howto-setup-dev-workstation]: /doc/contributor/howto-guide-setup-development-workstation.md
+[quickstart-link]: https://cloud.google.com/publicca/docs/quickstart
```
Comment (dbolduc) [inline google/cloud/publicca/quickstart/README.md:154]: there is (sadly) still a link here which 404s

File: google/cloud/publicca/quickstart/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+   ```
+
+   Note that Bazel automatically downloads and compiles all dependencies of the
+   project. As it is often the case with C++ libraries, compiling these
+   dependencies may take several minutes.
+
+1. Run the example, changing the placeholder(s) to appropriate values:
+
+   ```bash
+   bazel run :quickstart -- [...]
```
Comment (dbolduc) [inline google/cloud/publicca/quickstart/README.md:76]: nit: s/.../REGION/ ?

File: google/cloud/publicca/quickstart/README.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+   ```bash
+   cd $HOME/google-cloud-cpp/google/cloud/publicca/quickstart
+   cmake -S . -B .build -DCMAKE_TOOLCHAIN_FILE=$HOME/vcpkg/scripts/buildsystems/vcpkg.cmake
+   cmake --build .build
+   ```
+
+1. Run the example, changing the placeholder(s) to appropriate values:
+
+   ```bash
+   .build/quickstart [...]
```
Comment (dbolduc) [inline google/cloud/publicca/quickstart/README.md:113]: same

**Review by coryan — APPROVED** [review]

**Review by dbolduc — COMMENTED** [review]
I was too late.

**Pull request merged**